.ci/validation.sh

```diff
@@ -88,9 +88,14 @@ eclipse-static-analysis)
   ;;
 
 nondex)
-  # Below we exclude test that fails due to picocli library usage
-  mvn -e --no-transfer-progress --fail-never clean nondex:nondex -DargLine='-Xms1024m -Xmx2048m' \
-    -Dtest=!JavadocPropertiesGeneratorTest#testNonExistentArgument
+  # Exclude test that fails due to picocli library usage
+  SKIPPED_TESTS='!JavadocPropertiesGeneratorTest#testNonExistentArgument,'
+  # Exclude test that fails due to stackoverflow error
+  SKIPPED_TESTS+='!SingleSpaceSeparatorCheckTest#testNoStackoverflowError'
```

Comment:
I do not see a lot of value in execution of this test in nondex, and a quick investigation yields no obvious reason why this is failing. We can open an issue with them, but seems like the project is not very active.